templates/includes/sidebar.html

```diff
@@ -9,7 +9,7 @@
   <br>
 
   <li class="list-group-item"><a href="{% url 'taxi:index' %}">Home</a></li>
-  <li class="list-group-item"><a href="{% url 'taxi:driver-list' %}">All drivers</a></li>
-  <li class="list-group-item"><a href="{% url 'taxi:car-list' %}">All cars</a></li>
-  <li class="list-group-item"><a href="{% url 'taxi:manufacturer-list' %}">All manufacturers</a></li>
+  <li class="list-group-item"><a href="{% url 'taxi:driver_list' %}">All drivers</a></li>
+  <li class="list-group-item"><a href="{% url 'taxi:car_list' %}">All cars</a></li>
```

Comment:
Code Style issue: According to the checklist, you should use `-` to split words in URL identification parameter `name`, not the `_`. So, it should be `car-list` instead of `car_list`.